project/src/PlayerController.tscn

```diff
@@ -54,42 +57,98 @@ tracks/0/keys = {
 "update": 0,
 "values": [ false ]
 }
+tracks/1/type = "value"
+tracks/1/path = NodePath(".:hframes")
+tracks/1/interp = 1
+tracks/1/loop_wrap = true
+tracks/1/imported = false
+tracks/1/enabled = true
+tracks/1/keys = {
+"times": PoolRealArray( 0 ),
+"transitions": PoolRealArray( 1 ),
+"update": 0,
+"values": [ 6 ]
+}
+tracks/2/type = "value"
+tracks/2/path = NodePath(".:frame")
+tracks/2/interp = 1
+tracks/2/loop_wrap = true
+tracks/2/imported = false
+tracks/2/enabled = true
+tracks/2/keys = {
+"times": PoolRealArray( 0 ),
+"transitions": PoolRealArray( 1 ),
+"update": 0,
+"values": [ 0 ]
+}
+
+[sub_resource type="Animation" id=7]
+resource_name = "RunLeft"
+
+[sub_resource type="Animation" id=6]
+resource_name = "RunRight"
+length = 0.6
+loop = true
+step = 0.05
+tracks/0/type = "value"
+tracks/0/path = NodePath(".:frame")
+tracks/0/interp = 1
+tracks/0/loop_wrap = true
+tracks/0/imported = false
+tracks/0/enabled = true
+tracks/0/keys = {
+"times": PoolRealArray( 0, 0.1, 0.2, 0.3, 0.4, 0.5 ),
+"transitions": PoolRealArray( 1, 1, 1, 1, 1, 1 ),
+"update": 1,
+"values": [ 0, 1, 2, 3, 4, 5 ]
+}
 
 [sub_resource type="CapsuleShape2D" id=1]
 radius = 5.0
 height = 8.0
 
 [node name="PlayerController" type="KinematicBody2D" groups=["player"]]
 script = ExtResource( 1 )
+speed = 100
```

Comment:
@loteque @Anticdope The speed change surprises me. What was the purpose? Was this to match the sprite leg movement speed? If so, we might want to actually tweak the animation rather than the player speed. This speed bump results in the player moving at the same speed as the projectile which looks a bit odd and makes it trivial to avoid enemies.  What do you think?

Here are 2 sample videos:

https://user-images.githubusercontent.com/23508546/220987365-3b26968c-e1fa-40a9-af3c-803204989ad3.mp4

https://user-images.githubusercontent.com/23508546/220988894-149c4315-cc8f-4474-b8e4-2dc72c859e4f.mp4

Reply:
Lets adjust enemy movememt and bullets. The bullets are too slow imo as are the enemys.

Reply:
Also this branch was deleted and the associated issue was closed. Please open a new issue so your suggestions aren't lost. I'm closing this for now.

Reply:
@loteque What was the purpose of the speed change? I'm not trying to open an issue yet, just understand what's going on in the PR. If we have a task we can put in writing from discussion, I can make it and link it here.

Reply:
The speed change is because Antic and I determined that everything felt to slow.

I just want you to be aware that this PR has been merged.

Reply:
@jonathandavidlewis I agree with @loteque and we did consider these questions when making that speed change. Tweaking the animation is not the ideal solution as everything needed to be faster on screen anyways. We felt that projectile speed needed an increase, therefore bumping up player speed seemed trivial.

Reply:
Okay, thanks @loteque @Anticdope It sounds like you wanted to make more substantial changes to the dynamics of the game. Let's see if we can align on what that might look like. It was just surprising for me to experience that without the explanation.  Without that context, the only thing I could think of was that the legs in the animation were faster, so you bumped up the speed of the character. Thanks for understanding. Yeah, no worries about the PR being merged. I just wanted to have the context of that code and the PR for the convo, even though it's post-merge.